x-pack/plugins/triggers_actions_ui/public/application/hooks/use_load_alert_summary.ts

```diff
@@ -12,19 +12,10 @@ import { AsApiContract } from '@kbn/actions-plugin/common';
 import { HttpSetup } from '@kbn/core/public';
 import { BASE_RAC_ALERTS_API_PATH } from '@kbn/rule-registry-plugin/common/constants';
 import { useKibana } from '../../common/lib/kibana';
-
-export interface AlertSummaryTimeRange {
```

Comment:
Moved these types to `types.ts` file to export them from the plugin and use it on the Alerts page.